### PR TITLE
Postgresql support

### PR DESCRIPTION
**Pull request by verrydtj**

### CODE REVIEW

File: files/database/postgresql/module.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
```
Comment (treydock) [inline files/database/postgresql/module.xml]: I don't think this file is necessary, looks like your using the template and not this file.

**Comment by treydock** [comment]
Would be good to add example of how you call `keycloak` class to add postgresql support, like example of postgresql specific parameters necessary to add postgresql support.  Just add it under existing mysql example.

**Comment by verrydtj** [comment]
Indeed, I will add this next week and also remove that old module.xml file.?

________________________________

**Comment by treydock** [comment]
Released as 3.7.0

**Pull request merged**